test/unit/partials/mock-store/corrupted-package/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "corrupted-package"
```

Comment:
comma `,` missed.

Reply:
😝 ... it's a corrupted package, that's the point hehe ..

Reply:
shit.. I totally missed the name =D